recipe/meta.yaml

```
@@ -24,6 +26,9 @@ requirements:
     - llvm  # [llvmdev not in ("5.*", "6.*", "7.*")]
     - llvmdev
     - clangdev
+  run_constrained:
+    - cling ==9999999999  # [clangdev != "5.*"]
+    - cling 0.8           # [clangdev == "5.*"]
 
 test:
   commands:
```

Can you add a test like `xeus-cling` has to make sure the kernel is working: https://github.com/conda-forge/xeus-cling-feedstock/blob/master/recipe/run_test.py
```suggestion
  requires:                # [clangdev == "5.*"]
    - jupyter_kernel_test  # [clangdev == "5.*"]
  commands:
    - python $RECIPE_DIR/jupyer_kernel_test.py  # [clangdev == "5.*"]
```
(I'm suggesting `jupyer_kernel_test.py` instead of `run_test.py` so that it isn't ran automatically and you can enable it only for the clang 5 based build.)

---

Thank you!

---

Wouldn't this test need xeus-cling to be set as a requirement?

---

Yes it would need to be a test-only dependency like `jupyter_kernel_test`